### PR TITLE
Move signature block into the content of the consent.

### PR DESCRIPTION
**Pull request by alxdarksage**
See: https://sagebionetworks.jira.com/browse/BRIDGE-769

The signature block can now be changed by end users, the labels can be localized, etc. As long as they leave the name and date somewhere in the document (otherwise, well add the signature block to the end of the document). This will change existing templates as well so no migration is needed.

There is also a very minor change to the integration tests once this has been changed: https://github.com/Sage-Bionetworks/BridgeIntegrationTests/pull/378

### CODE REVIEW

File: src/main/java/org/sagebionetworks/bridge/services/StudyConsentService.java [hunk 1]
```diff
@@ -271,23 +296,27 @@ private String loadDocumentContent(StudyConsent consent) {
...
-        return document.body().html();
+    private String appendSignatureBlockIfNeeded(String content) {
+        // The user can change the signature block, they can remove parts of the signature block,
+        // but if the person's name or the signing date are gone, we're assuming that they've 
+        // removed too much, and we re-append the signature block.
+        if (!content.contains("${participant.name}") || !content.contains("${participant.signing.date}")) {
+            content = content + SIGNATURE_BLOCK;
+        }
+        return content;
     }
```
Comment (DwayneJengSage) [inline src/main/java/org/sagebionetworks/bridge/services/StudyConsentService.java:315]: Do we have any unit tests for where if the signature block already exists, we don't add another signature block?

File: src/test/java/org/sagebionetworks/bridge/services/ConsentPdfTest.java [hunk 2]
```diff
@@ -207,21 +184,13 @@ private static ConsentSignature makeInvalidSignature() {
...
-        assertTrue(bodyContent.contains(dateStr), "Signing date correct");
-        assertTrue(bodyContent.contains("|Test Person|"), "Name correct");
-        assertTrue(bodyContent.contains("|user@user.com|"), "User email correct");
-        assertTrue(bodyContent.contains("|Not Sharing|"), "Sharing correct");
-        assertTrue(bodyContent.contains("<html><head></head><body>Passed through as is."), "HTML markup preserved");
-    }
-
-    private static void validateNewDocBody(String bodyContent) throws Exception {
+    private static void validateDocBody(String bodyContent) throws Exception {
+        System.out.println(bodyContent);
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/services/ConsentPdfTest.java]: Minor: Remove println

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]
Some minor comments.

File: src/main/java/org/sagebionetworks/bridge/services/StudyConsentService.java (hunk 1 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/services/StudyConsentService.java:315]: Added

File: src/test/java/org/sagebionetworks/bridge/services/ConsentPdfTest.java (hunk 2 above)
Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/services/ConsentPdfTest.java]: removed

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**